### ISSUE TITLE
[Bug]: the perf of point_select of the same case are very different in the same distributed mo cluster at different time

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#9680]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 961aa4e
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

At first, after initiating a mo cluster, and run the sysbench oltp point_select for 10-tables-100000-per-table mode, the perf data is:
<img width="559" alt="image" src="https://github.com/matrixorigin/matrixone/assets/47846308/24fac230-4766-437c-9bc6-542900b14c3d">

and the index data is normal. But after cn pod restarted, and then run the same test, the tps data decrease seriously.
<img width="640" alt="image" src="https://github.com/matrixorigin/matrixone/assets/47846308/0e09a613-c90f-4b09-b8df-c5bc9ff650b5">

the profile of normal index:
[normal.zip](https://github.com/matrixorigin/matrixone/files/11562489/normal.zip)

the profile of exceptional index:
[slow.tar.gz](https://github.com/matrixorigin/matrixone/files/11562474/slow.tar.gz)
  

### Expected Behavior

The TPS of two times should be the same.

**Comment by zengyan1** [comment]
定位问题中，两个profile几乎没有差别，无法从中发现问题

**Comment by zengyan1** [comment]
正在看重启cn后的log

**Comment by zengyan1** [comment]
定位问题中

**Comment by zengyan1** [comment]
最新main已修复该bug

**Comment by aressu1985** [comment]
fixed:nightly-d4d74e7
